pkg/test/integration/camget_test.go

```diff
@@ -67,8 +66,9 @@ func TestCamgetSymlink(t *testing.T) {
 	}
 
 	out := test.MustRunCmd(t, w.Cmd("pk-put", "file", srcDir))
-	// TODO(mpl): rm call and delete pkg.
-	asserts.ExpectBool(t, true, out != "", "pk-put")
+	if out == "" {
+		t.Errorf("pk-put: expected output to be non-empty")
```

Comment:
Fatalf if you can't proceed (as it appears you can't here).

only use Errorf if the test can usefully continue and maybe report more problems without crashing

Audit the rest too?

Reply:
Good catch I'll audit the rest more carefully

Reply:
Ok i went through the changes and converted most of them to `Fatalf`. There are a few cases where multiple things are tested in the `jsonsign` test where its reasonable to continue on a bad assertions i think